src/display/canvas.js

```diff
@@ -1275,6 +1280,16 @@ class CanvasGraphics {
     this.cachedCanvases.clear();
     this.cachedPatterns.clear();
 
+    for (const cache of this._cachedBitmapsMap.values()) {
+      for (const canvas of cache.values()) {
+        if (canvas instanceof HTMLCanvasElement) {
```

Comment:
This code-path can be taken in Node.js environments as well, right?
In that case I don't believe that `HTMLCanvasElement` exists, which would throw an Error here and break things. Hence please add a `typeof HTMLCanvasElement !== undefined` check to avoid any problems.